### PR TITLE
[5.3] Artisan without mandatory arguments should output usage

### PR DESCRIPTION
**Pull request by riyuk**
PR for Issue #11878
PR for Master instead of 5.2 #13804

### CODE REVIEW

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 1]
```diff
@@ -113,6 +114,12 @@ public function handle($input, $output = null)
 
             $this->renderException($output, $e);
 
+            if (! $input->hasParameterOption(['--help', '-h'], true) && $input->hasArgument('command')) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/Kernel.php]: We should probably only look for `--help` here, since we're then calling `--help`, not `-h`.

**Comment by GrahamCampbell** [comment]
Other than my minor comment, this looks pretty good. :+1:

**Comment by riyuk** [comment]
Changed that

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**